l3experimental/l3pdf/l3pdf.dtx

```diff
@@ -174,6 +174,27 @@
 %   This function may only be used up to the point where the PDF file is
 %   initialised.
 % \end{function}
+% \subsection{Destinations}
+% % \begin{function}[added = 2021-01-03]
+%   {\pdf_destination:nn}
+%   \begin{syntax}
+%     \cs{pdf_destination:nn} \Arg{name} \Arg{type or integer}
```

Comment:
The 'keyword or scale' interface makes sense at the backend level, as it's essentially how the backends themselves work. However, I'm not so sure about this as a code-level interface.

Reply:
Imho it is fine. The hyperref interface is more complicated, but actually only because it tries to have similar options for pdfview, pdfstartview and pdfremotestartview. If I had only `pdfview` to consider (and wouldn't have to support an existing interface) I would  use 'keyword or scale' too.  It is a command quite near to the backend, but I don't see a need for more abstraction here.